venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,344 @@
+                    "authorship": note.content.get("authorship"),
+                    "paper_version": note.content.get("paper version"),
+                    "anonymity_period": note.content.get("anonymity period"),
+                    "commitment_note": f"https://openreview.net/forum?id={acl_commitment_note.forum}"
+                }  
+            )
+            acl_submitted = client.post_note(acl_sub)
+            if(acl_submitted):
+                submission_output_dict[acl_commitment_note.forum]['was_migrated'] = True
+                acl_submission_dict[acl_submitted.content['paper_link'].split('=')[1]]=acl_submitted
+                post_blind_submission(acl_submitted.id, acl_submitted, client.get_note(original_arr_sub_id), submission_output_dict, acl_commitment_note)
+            else: 
+                print(f'Nothing posted for ARR Submission {arr_submission_forum}')
+                return(submission_output_dict)
+        else:
+            print(f"signature of acl commitment note {acl_commitment_note.id} not in arr submission {arr_submission_forum} authors")
+            return submission_output_dict
+    else:
+        print(f"acl commitment note {acl_commitment_note.id} links to arr submission {arr_submission_forum} with invalid invitation")
+        return submission_output_dict
+            
+        
+    
+        
+# Post blind submission (calls post_reviews)
+def post_blind_submission(acl_submission_id, acl_submission, arr_submission, submission_output_dict, acl_commitment_note):
+    # Post requisite groups 
+    number = acl_submission.number
+    # Create paperX/Authors 
+    authors = openreview.Group(
+        id = 'aclweb.org/ACL/2022/Conference/Paper{number}/Authors'.format(number = number), 
+        signatures = [
+            'aclweb.org/ACL/2022/Conference'
+            ],
+        signatories = [
+            'aclweb.org/ACL/2022/Conference', 
+            'aclweb.org/ACL/2022/Conference/Paper{number}/Authors'.format(number = number)
+            ],
+        readers = [
+            'aclweb.org/ACL/2022/Conference', 
+            'aclweb.org/ACL/2022/Conference/Paper{number}/Authors'.format(number = number)
+            ],
+        writers = [
+            'aclweb.org/ACL/2022/Conference'
+            ],
+        members = acl_submission.content['authorids']
+    )
+
+    authors_posted = client.post_group(authors)
+    assert authors_posted, print('Failed to post author groups: ', acl_submission_id)
+
+    # Create paperX/Conflicts
+    conf_id = arr_submission.invitation.rsplit('/', 2)[0]
+    arr_number = arr_submission.number
+    conflict_members = [
+            '{conf_id}/Paper{number}/Reviewers'.format(conf_id = conf_id, number = arr_number),
+            '{conf_id}/Paper{number}/Area_Chairs'.format(conf_id = conf_id, number = arr_number),
+            'aclweb.org/ACL/2022/Conference/Paper{number}/Authors'.format(number = number)
```

Comment:
that means the authors can not see their own submissions?

Reply:
I think this is what we originally agreed to do, but now we are making authors readers after the commitment deadline.